### ISSUE TITLE
Bug when saving null values in UC Grid

### ISSUE THREAD

**Issue by rimblas** [plug-ins-pro/enhanced_grid_pro#4]
This one seems pretty strange. 
To reproduce:
 * Double-click on a numeric cell used to calculate a different column.
 * Empty out the value
 * Click Save

The empty field gets a "garbage" value.

It seems to occur when the SQL query contains a calculated column.
For example:
```
select EMPNO,
       ENAME,
       SAL,
       COMM,
       DEPTNO
     , (nvl(SAL,1) * 2 * COMM) sub_total
  from EMP
```
You can see the AJAX request sends an empty string. But returns a large number.

<img width="1897" alt="image" src="https://github.com/plug-ins-pro/enhanced_grid_pro/assets/2160702/33bcd3ca-65e3-46c3-9fc0-7a0e44d726a2">

We created a one-page app with the grid and the SQL above.  Query Only and Read Only Always for the sub_total

<img width="891" alt="image" src="https://github.com/plug-ins-pro/enhanced_grid_pro/assets/2160702/c2cf592b-662c-42db-92cf-0c85739d4b00">

**Comment by andraz-podobnik** [comment]
Hi,

Even though we were in contact over the email, I'll leave a comment here too.

I wasn’t able to completely replicate your issue with getting a “garbage” value.
However, I noticed a weird behaviour in some cases – and made a change so that when you remove the value, it always takes it as null. Before, it was taken as empty string “” (also seen on the screenshots you attached).

The beta version I prepared fixed the issue (confirmed in the email) - this fix will be released with Enhanced Grid Pro version v23.2.
